misc/python/materialize/mzcompose/composition.py

```diff
@@ -124,7 +124,6 @@ def __init__(
             raise UnknownCompositionError(name)
 
         self.compose: dict[str, Any] = {
-            "version": "3.7",
```

Comment:
Should we set a more recent version instead for repeatability?

Reply:
The entire "version" field is obsolete, no matter what number you put in

Reply:
Ah, okay 😀